triton/whisper/export-onnx-triton.sh

```diff
@@ -0,0 +1,16 @@
+export CUDA_VISIBLE_DEVICES="0"
+
+model_repo_path=./model_repo_whisper
+
+encoder_fp32_file=./large-v2-encoder.onnx
```

Comment:
Could you use
```suggestion
name=large-v2
model_repo_path=./model_repo_whisper_${name}

encoder_fp32_file=./${name}-encoder.onnx
# or
encoder_fp32_file=./encoder.onnx
```

In this way, it makes it easier to support other types of models, e.g., small, medium, etc.